index.js

```diff
@@ -1,21 +1,18 @@
 document.addEventListener("DOMContentLoaded", () =>{
-	let root = document.documentElement;
+	const root = document.documentElement;
 
 	// Character
 	let nameWritten = false;
 	let nameChar;
 	let classSelected;
 	let hp;
 	let hpMax;
-	let hpBar;
 	let mana;
 	let manaMax;
-	let manaBar;
 	let rage;
 	let rageMax;
 	let energy;
 	let energyMax;
-	let energyBar;
 	
```

Comment:
All three above are not being used anywhere in the code. My editor told me so - these variables should be removed and when you need them in the future, add them again. This removes code bloatness.